WcaOnRails/app/models/competition.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
@jonatanklosko, this actually isn't a rubocop comment. This is a comment that tells Ruby 2.3 to treat all string allocations in this file as immutable strings. @pingskills, I believe the fact that you added this comment means you can actually remove the call to `.freeze` on line 52.

The rubocop @pingskills ran into appears to be rubocop helping force us to prepare for Ruby 3.0 where strings will be immutable by default. Here's what I see if I run rubocop on @pingskills's project after removing this comment:

```
~/gitting/worldcubeassociation.org/WcaOnRails @kaladin> bundle exec rubocop
Inspecting 279 files
.................................................................................................................................................C.....................................................................................................................................

Offenses:

app/models/competition.rb:1:1: C: Missing frozen string literal comment.
class Competition < ActiveRecord::Base
^

279 files inspected, 1 offense detected
```

I support this comment. In a later PR, I'd like to turn this on everywhere. If we make use of immutable strings anywhere, I'm going to be rather surprised.